src/main/java/de/tum/in/www1/artemis/web/rest/dto/LectureUnitSplitDTO.java

```diff
@@ -0,0 +1,49 @@
+package de.tum.in.www1.artemis.web.rest.dto;
+
+import java.time.ZonedDateTime;
+
+public class LectureUnitSplitDTO {
+
+    public String unitName;
+
+    public ZonedDateTime releaseDate;
+
+    public String startPage;
```

Comment:
It is a little bit unclear for me why this is a "String"? Isnt it a pagenumber?

Reply:
Yes you are right it should be Integer. Changed to Integer.